### PR TITLE
Feature/films list

### PR DESCRIPTION
**Pull request by ViktorVikVik**

### CODE REVIEW

File: src/components/FilmsList.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  height: fit-content;
+  background-color: white;
+  border: 1px solid;
+  padding: 5px;
+}
+.leftSide {
+  display: inline;
+  float: left;
+}
+.rightSide {
```
Comment (zayacny) [inline src/components/FilmsList.css]: Look for in Google "how to name classes in CSS"

File: src/components/FilmsList.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./FilmsList.css";
+
+const films = [
```
Comment (zayacny) [inline src/components/FilmsList.js]: Add this const to App.js and pass it to your <FilmsList> component. It's mentioned in task. The component should get props(parametr) as array of objects.

File: src/components/FilmsList.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  {
+    name: "turtles",
+    year: 1990,
+    link: "http://baskino.me/films/boevie-iskustva/9830-cherepashki-nindzya.html",
+    director: "M. Jackson",
+    rating: 8.88,
+  },
+];
+
+export default function FilmsList() {
```
Comment (zayacny) [inline src/components/FilmsList.js:4]: This function have to get films array as parametr.

File: src/components/FilmsList.css (hunk 1 above)
Comment (lixolitr) [reply src/components/FilmsList.css]: right-side
we use in CSS kebab-case, not camelCase (Java, JS, ...)

**Review by zayacny — APPROVED** [review]

**Comment by lixolitr** [comment]
Ok

**Review by lixolitr — APPROVED** [review]

**Pull request merged**